src/postman/transformers/request.ts

```diff
@@ -64,8 +64,8 @@ export function transformBody(body: RequestBody, mediaType?: string): IHttpOpera
   return;
 }
 
-function transformRawBody(raw: string, mediaType: string = 'text/plain'): IMediaTypeContent {
-  if (typeIs.is(mediaType, ['application/json', 'application/*+json'])) {
+export function transformRawBody(raw?: string, mediaType: string = 'text/plain'): IMediaTypeContent {
```

Comment:
If there's no body provided, the caller should just not call this function at all, why make the most important argument optional?

Reply:
Done 👍